docs/content/envoy-primer.md

```diff
@@ -0,0 +1,347 @@
+---
+title: Policy Primer via Examples
+kind: envoy
+weight: 2
+---
+
+This page covers how to write policies for the content of the requests that are passed to OPA by Envoy's
+[External Authorization
+filter](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/security/ext_authz_filter.html).
+
+## Writing Policies
+
+Let's start with an example policy that restricts access to an endpoint based on a user's role and permissions.
+
+```live:bool_example:module:read_only
```

Comment:
Let's make it so that this example (and the object one below) take advantage of the playground. Make this example openable and below show some simplified input and policy evaluation results.

Reply:
Both examples are openable.